### DISCUSSION TITLE
Unstyled PrimeVue with Nuxt

### DISCUSSION THREAD

**Discussion by samdickerman** [primefaces/community#464] [PrimeVue] [answered]
Hello,

Can you please post an example of how to use the cut and paste method for using tailwind with unstyled PrimeVue?  The [example in the docs](https://primevue.org/tailwind/) is for Vue in the app.js file.

Thanks!

Sam

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello,

You could follow these steps [here](https://primevue.org/tailwind/#setup) and copy-paste the desired component style. For example Checkbox https://primevue.org/checkbox/#theming.tailwind
